processor/geoipprocessor/internal/provider/maxmindprovider/provider_test.go

```diff
@@ -19,10 +20,14 @@ import (
 
 func TestInvalidNewProvider(t *testing.T) {
 	_, err := newMaxMindProvider(&Config{})
-	require.ErrorContains(t, err, "could not open geoip database: open : no such file or directory")
+	expectedErrMsgSuffix := "no such file or directory"
+	if runtime.GOOS == "windows" {
+		expectedErrMsgSuffix = "The system cannot find the file specified."
+	}
+	require.ErrorContains(t, err, "could not open geoip database: open : " + expectedErrMsgSuffix)
 
 	_, err = newMaxMindProvider(&Config{DatabasePath: "no valid path"})
-	require.ErrorContains(t, err, "could not open geoip database: open no valid path: no such file or directory")
+	require.ErrorContains(t, err, "could not open geoip database: open no valid path: " + expectedErrMsgSuffix)
```

Comment:
The two `require` statements here are failing lint:
```
geoipprocessor/internal/provider/maxmindprovider/provider_test.go:27: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/open-telemetry/opentelemetry-collector-contrib) (gci)
	require.ErrorContains(t, err, "could not open geoip database: open : " + expectedErrMsgSuffix)
geoipprocessor/internal/provider/maxmindprovider/provider_test.go:30: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/open-telemetry/opentelemetry-collector-contrib) (gci)
	require.ErrorContains(t, err, "could not open geoip database: open no valid path: " + expectedErrMsgSuffix)
```

Reply:
ee1ab69321